src/fdcache.cpp

```diff
@@ -93,7 +91,7 @@ pthread_mutex_t FdManager::fd_manager_lock;
 pthread_mutex_t FdManager::cache_cleanup_lock;
 pthread_mutex_t FdManager::reserved_diskspace_lock;
 bool            FdManager::is_lock_init(false);
-string          FdManager::cache_dir;
+std::string          FdManager::cache_dir;
```

Comment:
If possible, please align the indentation.

Reply:
Done.